build_tools/ci/cpu_comparison/run.py

```diff
@@ -1267,22 +1124,41 @@ def benchmark_aie_kernel_time(
     n_repeats,
     n_kernel_runs,
     n_reconfigure_runs,
+    n_pdi_loads,
     time_unit,
 ):
     """
     Benchmark a compiled AIE module's (aie_vmfb) kernel time, average over the specified number of runs.
     """
     test_dir = config.get_test_dir(name)
-    aie_bin = test_dir / f"{name}_aie.bin"
 
-    if n_kernel_runs > 0 and n_reconfigure_runs > 0:
+    if n_pdi_loads <= 0:
         raise ValueError(
-            "Cannot set both n_kernel_runs and n_reconfigure_runs simultaneously. "
-            "This will produce incorrect performance results. "
-            "To measure kernel execution time, set n_kernel_runs > 1 and n_reconfigure_runs == 0. "
-            "To measure reconfiguration time, set n_kernel_runs == 0 and n_reconfigure_runs > 0."
+            "PDI needs to be loaded at least once for the benchmark to work. "
+            f"Got n_pdi_loads={n_pdi_loads}."
+        )
+
+    """
+    Check consistency only for values > 0 (or >1 for PDI loads)
+    Valid Examples:
+    - n_kernel_runs=0, n_reconfigure_runs=0, n_pdi_loads=10 (measure PDI load time)
+    - n_kernel_runs=10, n_reconfigure_runs=0, n_pdi_loads=1 (measure kernel time)
+    - n_kernel_runs=0, n_reconfigure_runs=10, n_pdi_loads=1 (measure reconfigure time)
+    - n_kernel_runs=10, n_reconfigure_runs=10, n_pdi_loads=1 (measure kernel and reconfigure time)
+    """
```

Comment:
Prefer `#` for block comments: https://peps.python.org/pep-0008/#block-comments
```suggestion
    # Check consistency only for values > 0 (or > 1 for PDI loads)
    # Valid Examples:
    # - n_kernel_runs=0, n_reconfigure_runs=0, n_pdi_loads=10 (measure PDI load time)
    # - n_kernel_runs=10, n_reconfigure_runs=0, n_pdi_loads=1 (measure kernel time)
    # - n_kernel_runs=0, n_reconfigure_runs=10, n_pdi_loads=1 (measure reconfigure time)
    # - n_kernel_runs=10, n_reconfigure_runs=10, n_pdi_loads=1 (measure kernel and reconfigure time)
```

Reply:
Done.